agent/security_helper.py

```diff
@@ -336,3 +340,32 @@ def cpu_vulnerabilities():
         res['mitigations_disabled'] = mitigations_disabled
 
     return res
+
+
+def patch_sshd_config(patch_param, safe_value):
+    if not os.path.isfile(SSHD_CONFIG_PATH):
+        return
+    with open(SSHD_CONFIG_PATH, 'r+') as sshd_config:
+        replaced = False
+        lines = sshd_config.readlines()
+        patched_lines = []
+        for line in lines:
+            patched_lines.append(line)
+            line = line.strip()
+            if not line or line[0] == '#':
+                # skip empty lines and comments
+                continue
+            line_split = line.split(maxsplit=1)
+            if len(line_split) != 2:
+                continue
+            param, value = line_split
+            value = value.strip('"')
+            if param == patch_param and value != safe_value:
+                logger.info('{}: replacing "{}" with "{}"'.format(param, value, safe_value))
+                patched_lines[-1] = param + ' ' + safe_value + '\n'
+                replaced = True
+        if replaced:
+            shutil.copy(SSHD_CONFIG_PATH, SSHD_CONFIG_PATH + '.bup')
```

Comment:
Can we copy this file to `/opt/wott/backups/sshd_config.$TIMESTAMP` or similar instead?